src/pages/editor/curators/AddCuratorDialog.tsx

```diff
@@ -0,0 +1,180 @@
+      viewingScope = viewingScope.filter(
+        (id) => !parentOrganizationIds.includes(id) && !childOrganizationIds.includes(id)
+      );
+
+      if (newOrganizationId) {
+        viewingScope.push(newOrganizationId);
+      }
+
+      const initialValues: InstitutionUser = {
+        ...currentUser,
+        viewingScope: {
+          ...currentUser.viewingScope,
+          includedUnits: viewingScope,
+        },
+      };
+
+      setUserInitialValues(initialValues);
+    }
+  }, [currentOrganization, userQuery.data]);
+
+  return (
+    <Dialog open={open} onClose={closeDialog} maxWidth="md" fullWidth>
+      <DialogTitle>{t('editor.curators.add_curator')}</DialogTitle>
+
+      <DialogContent>
+        <Autocomplete
+          options={employeeSearchQuery.data?.hits ?? []}
+          renderOption={(props, option) => (
+            <li {...props} key={option.id}>
+              {getFullCristinName(option.names)}
+            </li>
+          )}
+          onInputChange={(_, value, reason) => {
+            if (reason === 'clear' || reason === 'reset') {
+              setSearchQuery('');
+              setSelectedPerson(null);
+              setUserInitialValues(null);
+            } else {
+              setSearchQuery(value);
+            }
+          }}
+          onChange={async (_, value) => {
+            setSearchQuery('');
+            setSelectedPerson(value);
+          }}
+          getOptionLabel={(option) => getFullCristinName(option.names)}
+          isOptionEqualToValue={(option, value) => option.id === value.id}
+          loading={employeeSearchQuery.isFetching}
+          sx={{ mb: '1rem' }}
+          renderInput={(params) => (
+            <AutocompleteTextField
+              // data-testid={dataTestId.registrationLandingPage.tasksPanel.assigneeSearchField}
+              {...params}
+              label={t('common.person')}
+              isLoading={employeeSearchQuery.isFetching}
+              placeholder={t('common.search')}
+              showSearchIcon
+            />
+          )}
+        />
+
+        {selectedPerson && (
+          <>
+            {userQuery.isLoading || (userQuery.data && !userInitialValues) ? (
+              <CircularProgress />
+            ) : userQuery.data && userInitialValues ? (
+              <AddCuratorForm
+                closeDialog={closeDialog}
+                initialValues={userInitialValues}
+                currentUser={userQuery.data}
+                refetchCurators={refetchCurators}
+              />
+            ) : (
+              <>
+                <Typography>
+                  Brukeren du ønsker å legge til som kurator må logge inn i løsningnen for å få tildelt en bruker før du
+                  kan gi dem kuratortilgang.
+                </Typography>
+                <Typography>
+                  Be {getFullCristinName(selectedPerson.names)} om å logge inn i NVA før du prøver igjen.
+                </Typography>
```

Comment:
Dette skal vekk i neste PR, hvor vi også skal ha støtte for å opprette brukere.
Lar derfor dette ligge som norsk, sånn at oversetter ikke sløser tid på å fikse på andre språk også